docs/plan

```diff
@@ -0,0 +1,57 @@
+Step 1 — Test the prices_map shape for TCGplayer (planning only)
```

Comment:
when we talk we should talk about what this is and why its here and if we're going to keep it. if we keep it then it should make sense and be numbered.